test/e2e/far_e2e_test.go

```diff
@@ -0,0 +1,208 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+	farController "github.com/medik8s/fence-agents-remediation/controllers"
+	"github.com/medik8s/fence-agents-remediation/pkg/cli"
+	farUtils "github.com/medik8s/fence-agents-remediation/test/e2e/utils"
+)
+
+const (
+	// testNamespace  = "fence-agents-remediation"
+	testNamespace  = "openshift-operators"
+	fenceAgentIPMI = "fence_ipmilan"
+
+	// eventually parameters
+	timeout      = 2 * time.Minute
+	pollInterval = 10 * time.Second
+	offsetExpect = 1
```

Comment:
I can't follow. What's the improvement here? Why would writing `offsetOfOneForExpect` instead of `1` be better? 🤔 

When we go further down that road we end up with
```
const (
    ONE = 1
    TWO = 2
    THREE = 3

    offsetOfOneForExpect = ONE
)

func add(a, b int) (int, error) {
    if a == ONE && b == 2 {
       return THREE, nil
    }
    return 0, fmt.Errorf("running out of  \"MACROS\" for integers!")
}
```
😉